src/quickwrench_api/apps/workshops/serializers.py

```diff
@@ -0,0 +1,53 @@
+from typing import Iterable
+
+from rest_framework import serializers
+
+from ..accounts.serializers import AccountSerializer
+from .models import Account, Category, Workshop, Service
+
+
+class CategorySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Category
+        fields: Iterable[str] = (
+            "name",
+            "description",
+        )
+
+
+class ServiceSerializer(serializers.ModelSerializer):
+    category = serializers.PrimaryKeyRelatedField(queryset=Category.objects.all())
+
+    class Meta:
+        model = Service
+        fields: Iterable[str] = (
+            "category",
+            "name",
+            "description",
+            "price",
+        )
+
```

Comment:
You need to add a new endpoint (`/workshops/categories/`) for getting all available service categories and their ids in order for the frontend to know how to reference a category.